src/Controller/CategoryAdminController.php

```diff
@@ -27,12 +29,15 @@
  */
 class CategoryAdminController extends Controller
 {
-    /**
-     * @param Request $request
-     *
-     * @return Response
-     */
-    public function listAction(?Request $request = null)
+    public static function getSubscribedServices(): array
```

Comment:
I guess the controller should be defined as a service, something like https://github.com/sonata-project/SonataAdminBundle/blob/3214dec4699374e647e9e172a1043a71c273edad/src/Resources/config/core.php#L126-L129